### ISSUE TITLE
Formatting options

### ISSUE THREAD

**Issue by markusjohnsson** [jscheiny/safe-units#140]
This seams like a very useful library and I've been toying with it a bit. 

I was wondering if there are more formatting options, corresponding to toFixed and toPrecision. I was using the lib in a react application and wanted to some simple unit conversion for display, but I got this, which is not really suitable for display.

```jsx
<span>{distance.in(kilo(meters))}</span> // "29.023945563497445 km"
```

Is there another way to do this or is it a limitation?

**Comment by jscheiny** [comment]
Good note, I definitely meant to include the ability to format measures at some point but it slipped my mind. I have a concept for the API for this and will create a PR soon.

**Comment by markusjohnsson** [comment]
Great to hear!

Just a little note: I was fiddling with some formatting code myself and I quickly found that for any Time measures in the order of minutes and hours, I really prefer to have `HH:mm:ss` rather than decimal formatting. I don't know if that is part of your plan since it is specific for the Time dimension; but I think I might not be the only one wanting this.

**Comment by jscheiny** [comment]
Just opened up a PR, would love to hear your thoughts on the API.

With respect to time formatting I'm a little torn on if I think this library should handle it and if so, how. Will give it some more thought but its a good suggestion regardless.

**Comment by jscheiny** [comment]
This has been released in v1.1.0!
